rollup-plugins/production-pruning.js

```diff
@@ -63,24 +63,27 @@ const formatMessage = (node) => {
 
 const createErrorDocumentation = (node) => {
   const errorMessage = formatMessage(node);
-  const errorIndex = errors.push(errorMessage);
 
-  const markdownOutput = `
-## Error ${errorIndex}:
+  if (errors.includes(errorMessage)) {
+    return errors.indexOf(errorMessage);
+  } else {
+    const errorIndex = errors.push(errorMessage);
 
+    const markdownOutput = `
+## Error ${errorIndex}
 ${errorMessage}
-`;
+  `;
 
-  fs.appendFile('./docs/error.md', markdownOutput, (err) => {
-    if (err) throw err;
-  });
+    fs.appendFile('./docs/error.md', markdownOutput, (err) => {
+      if (err) throw err;
```

Comment:
I love single line conditionals but i think they are against our linting convention for the app. Can you add some brackets

Reply:
will do!